api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/MultiValueEditSubmission.java

```diff
@@ -172,20 +174,14 @@ public Literal createLiteral_ORIG(String value, String datatypeUri, String lang)
 	/* maybe this could be static */
 	public Literal createLiteral(String value, String datatypeUri, String lang) {
 		if( datatypeUri != null && !datatypeUri.isEmpty() ){
-			// UQAM Original code contained tow-dots ':' in place of '#'
-//            if( "http://www.w3.org/2001/XMLSchema:anyURI".equals(datatypeUri) ){
 			if( XSD.anyURI.getURI().equals(datatypeUri) ){
-//				try {
-//					return literalCreationModel.createTypedLiteral( URLEncoder.encode(value, "UTF8"), datatypeUri);
 					return literalCreationModel.createTypedLiteral( value, datatypeUri);
-//				} catch (UnsupportedEncodingException e) {
-//					log.error(e, e);
-//				}
-			} else if ( XSD.xstring.getURI().equals(datatypeUri) || RDF.dtLangString.getURI().equals(datatypeUri) ){
```

Comment:
Hi @litvinovg, could you help me understand why `XSD.xstring.getURI()` is no longer a valid condition here and below?